### ISSUE TITLE
container doesn't exist

### ISSUE THREAD

**Issue by ashwani29** [checkpoint-restore/criu#1144]
I'm trying to checkpoint a container with the command 

> sudo runc checkpoint <container_id>

It throws a error saying `container <conatiner_id> doesn't exist` whereas it is still running and checked with command `runc list`

what is the problem here?

**Comment by adrianreber** [comment]
Please provide all the command and all the output exactly as you used them. This sounds like you are doing something wrong.

**Comment by ashwani29** [comment]
i started the container as `sudo runc run ctr &>test.log && </dev/null/`
it has a single process which is just incrementing a counter variable and printing it.
then I tried to checkpoint it via `sudo runc checkpoint ctr` command.
I checked again the status of the container with `sudo runc list` and it showed the status as running. 
One strange thing was one container that was running earlier then i closed the terminal was not showing its status on the above command rather it's status is shown when I typed without `sudo prefix` and then another one i.e. ctr container's status is not shown.
